docs/en/observability/synthetics-troubleshooting.asciidoc

```diff
@@ -65,6 +65,33 @@ in the {synthetics-app}.
 
 To fix this, make sure there is an agent configured to run against the agent policy.
 
+[discrete]
+[[synthetics-troubleshooting-no-direct-es-connection]]
+== No results from a monitor
+
+If you have configured a monitor but don't see any results for that monitor in the {synthetics-app}, whether running them from Elastic's global managed testing infrastructure, or from {private-location}s, ensure Synthetics has a direct connection to {es}.
+
+Do not configure any ingest pipelines, or output via Logstash as this will prevent Synthetics from working properly and is not <<synthetics-support-matrix,supported>>.
+
+[discrete]
+[[synthetics-troubleshooting-no-agent-running]]
+== Browser monitor configured to run on a {private-location} not running to schedule
```

Comment:
The build is failing because this id is used above (probably a copy-paste error?).  Maybe something like this:


```suggestion
[discrete]
[[synthetics-troubleshooting-private-location-off-schedule]]
== Browser monitor configured to run on a {private-location} not running to schedule
```